app/org/sagebionetworks/bridge/json/ScheduleDeserializer.java

```diff
@@ -20,7 +23,7 @@
 import com.fasterxml.jackson.databind.DeserializationContext;
 import com.fasterxml.jackson.databind.JsonDeserializer;
 import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.collect.Lists;
+import com.fasterxml.jackson.databind.node.ArrayNode;
 
 public class ScheduleDeserializer extends JsonDeserializer<Schedule> {
```

Comment:
So I know Jackson can handle lists of things without a problem:
https://github.com/Sage-Bionetworks/BridgePF/blob/develop/app/org/sagebionetworks/bridge/dynamodb/DynamoUploadSchema.java#L41

So at most we need custom serializers for LocalTime, which should be easy to hook up in our custom module:
https://github.com/Sage-Bionetworks/BridgePF/blob/develop/app/org/sagebionetworks/bridge/json/BridgeObjectMapper.java#L57